packages/vue-app/template/server.js

```diff
@@ -47,13 +48,33 @@ const createNext = ssrContext => (opts) => {
 // Since data fetching is async, this function is expected to
 // return a Promise that resolves to the app instance.
 export default async (ssrContext) => {
+  // const _consoleLog = console.log
+  // const logs = []
+  // console.log = function (message, ...args) {
+  //   logs.push({ message, stack: this.stack})
+  //   _consoleLog(message, ...args)
+  // }
+  <% if (isDev) { %>
+  const logs = []
+  const devReporter = {
+    log(logObj) {
+      if (logObj.args[0] instanceof Error) {
+        logObj.args[0] = logObj.args[0].stack
```

Comment:
Nice check. Devalue will be broken without this.

Reply:
Yes I’ve got the warning when I tried without :)